docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       - "./scripts/postgres_init.sql:/docker-entrypoint-initdb.d/postgres_init.sql"
     restart: unless-stopped
   indexd-service:
-    image: "quay.io/cdis/indexd:master"
+    build: "../indexd/"
```

Comment:
should put back to image: "quay.io?cdis/indexd:master" /

Reply:
thanks, just fixed it :)